android/cli/lib/process-splashes-task.js

```diff
@@ -0,0 +1,51 @@
+const path = require('path');
+const CopyResourcesTask = require('../../../cli/lib/tasks/copy-resources-task');
+
+const appc = require('node-appc');
+const i18n = appc.i18n(__dirname);
+const __ = i18n.__;
```

Comment:
<!--
  0 failure: 
  1 warning:  android/cli/lib/p...
  
  
  DangerID: danger-id-Danger;
  File: android/cli/lib/process-splashes-task.js;
  Line: 6;
-->

- :warning: android/cli/lib/process-splashes-task.js line 6 – '__' is assigned a value but never used. (no-unused-vars)